spec/predicates/runtime-trace.md

```diff
@@ -80,49 +105,68 @@ Identifies the specific monitor instance used to trace the runtime.
 
 URI indicating the monitor’s identity.
 
-`build` _object_, _required_
+`process` _object_, _required_
 
-Feedback required: Should this predicate be scoped to runtime traces of builds only or generalize to runtime traces of any activity?
+`process.HostID` _string (URI)_, _required_
 
-`builder.builderId` _string (URI)_, _required_
+URI indicating the process host’s identity. Ex: GitLab Runner ID
 
-URI indicating the builder’s identity. Ex: GitLab Runner ID
+`process.type` _string (TypeURI)_, _required_
 
-`build.type` _string (TypeURI)_, _required_
+URI indicating the type of process performed.
 
-URI indicating the type of build performed.
-
-`build.event` _string_, _required_
+`process.event` _string_, _required_
 
 String identifying the specific job or task associated with the attestation.
 
 `invocation` _object_, _optional_
 
-Identifies the event that initiated the monitor. Similar to `invocation` in SLSA Provenance.
+Identifies the event that initiated the monitor.
```

Comment:
I almost wonder if `invocation` could be taken out and `invocation.configSource` moved under `process`? This may make even more sense if the `tracePolicy` is moved as suggested below.